sim/midas/src/main/scala/midas/passes/WriteXDCFile.scala

```diff
@@ -0,0 +1,102 @@
+// See LICENSE for license details.
+
+package midas.passes
+
+import firrtl._
+import firrtl.stage.Forms
+import firrtl.annotations._
+import firrtl.analyses.InstanceKeyGraph
+
+import midas.stage.{GoldenGateFileEmission}
+import midas.targetutils.xdc._
+
+/**
+  * We could reuse [[GoldenGateOutputFileAnnotation]] here, but this makes it
+  * marginally easier to filter out.
+  */
+private[midas] case class XDCOutputAnnotation(fileBody: String, suffix: Option[String])
+    extends NoTargetAnnotation with GoldenGateFileEmission {
+  def getBytes = fileBody.getBytes
+}
+
+private[midas] object WriteXDCFile extends Transform with DependencyAPIMigration with XDCAnnotationConstants {
+	override def prerequisites = Forms.LowForm
```

Comment:
```suggestion
    override def prerequisites = Forms.LowForm
```
What's with the tabs?